geoutils/geoviewer.py

```diff
@@ -1,98 +1,170 @@
+        default="default",
+        help="str, a matplotlib colormap string (default is from rcParams).",
+    )
+    parser.add_argument(
+        "-vmin",
+        dest="vmin",
+        type=str,
+        default=None,
+        help=(
+            "float, the minimum value for colorscale, or can be expressed as a "
+            "percentile e.g. 5%% (default is calculated min value)."
+        ),
+    )
+    parser.add_argument(
+        "-vmax",
+        dest="vmax",
+        type=str,
+        default=None,
+        help=(
+            "float, the maximum value for colorscale, or can be expressed as a "
+            "percentile e.g. 95%% (default is calculated max value)."
+        ),
+    )
+    parser.add_argument(
+        "-band",
+        dest="band",
+        type=int,
+        default=None,
+        help="int, which band to display (start at 0) for multiband images (Default is 0).",
+    )
+    parser.add_argument(
+        "-nocb",
+        dest="nocb",
+        help="If set, will not display a colorbar (Default is to display the colorbar).",
+        action="store_false",
+    )
+    parser.add_argument(
+        "-clabel", dest="clabel", type=str, default="", help="str, the label for the colorscale (Default is empty)."
+    )
+    parser.add_argument("-title", dest="title", type=str, default="", help="str, figure title (Default is empty).")
+    parser.add_argument(
+        "-figsize",
+        dest="figsize",
+        type=str,
+        default="default",
+        help=(
+            "str, figure size, must be a tuple of size 2, either written with quotes, "
+            "or two numbers separated by comma, no space (Default is from rcParams)."
+        ),
+    )
+    parser.add_argument(
+        "-max_size",
+        dest="max_size",
+        type=int,
+        default=2000,
+        help="int, image size is limited to max_size**2 for faster reading/displaying (Default is 2000).",
+    )
+    parser.add_argument(
+        "-save",
+        dest="save",
+        type=str,
+        default="",
+        help="str, filename to the output filename to save to disk (Default is displayed on screen).",
+    )
+    parser.add_argument(
+        "-dpi",
+        dest="dpi",
+        type=str,
+        default="default",
+        help="int, dpi value to use when saving figure (Default is from rcParams).",
+    )
+    parser.add_argument(
+        "-nodata",
+        dest="nodata",
+        type=str,
+        default="default",
+        help="float, no data value (Default is read from file metadata).",
+    )
+    parser.add_argument(
+        "-noresampl",
+        dest="noresampl",
+        default=False,
+        action="store_true",
+        help="True or False, if False then allow dynamic image downscaling, if True, prevent it.",
+    )
 
     args = parser.parse_args()
 
     return args
 
 
-def main():
+def main() -> None:
 
     # Parse arguments
     args = getparser()
 
-    ## Load image metadata ##
+    # Load image metadata #
```

Comment:
I like to use 2 or 3 # to delineate larger code blocks... e.g.:
```
## Read input data set ##

# Read input 1
...

# Read input 2
...

## Process data ##

# Calculation 1
...

# Calculation 2
...
```
What would be the alternative?

Reply:
I get it, and it's more or less personal style. I think it's `flake8` that complains about this. It's all about consistent decisions in the end!

Reply:
What do you use if you want to have a different comment "level" then?

Reply:
I don't know! I'm sure there's documentation on that somewhere. One way is to just do:
```python
#
#
# NEW SECTION
# 
#
```
which I think is fine.